src/scripts/main.coffee

```diff
@@ -32,6 +32,30 @@ class FormbuilderModel extends Backbone.DeepModel
     parent and parent.get('type') is 'grid'
   canBeConditionallyDisplayed: () -> !@inTable() and !@inGrid() and Formbuilder.conditionalFunctionality
   canShowReferenceID: () -> Formbuilder.showReferenceIDFunctionality
+
+  findConditionalParent: () ->
```

Comment:
Is the purpose of this function to return UUIDs of all ancestors up to the top level?
As the name of the function is findConditionalParent, it sounds like its job is to return the highest level FormbuilderModel ancestor that has conditional dependence.

Maybe it would be better to name it something like findConditionalAncestorUuids() ? Let me know if I'm misinterpreting this though.